<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab_Init</class>
 <widget class="QWidget" name="Tab_Init">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Substrate Molecule:</string>
     </property>
     <property name="buddy">
      <cstring>edit_substrateFile</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="edit_substrateFile"/>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Matrix Molecules:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="push_substrateBrowse">
     <property name="text">
      <string>&amp;Browse...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="push_substrateCurrent">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Use &amp;Current</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="push_matrixRemove">
     <property name="text">
      <string>&amp;Remove</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="push_matrixCurrent">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Add &amp;Current</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="push_matrixAdd">
     <property name="text">
      <string>&amp;Add</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QTableWidget" name="table_matrix">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>#</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Stoich</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Filename</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edit_substrateFile</tabstop>
  <tabstop>push_substrateBrowse</tabstop>
  <tabstop>push_substrateCurrent</tabstop>
  <tabstop>table_matrix</tabstop>
  <tabstop>push_matrixAdd</tabstop>
  <tabstop>push_matrixRemove</tabstop>
  <tabstop>push_matrixCurrent</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
